### DISCUSSION TITLE
[Feature] Arrange Images and Albums as you like

### DISCUSSION THREAD

**Discussion by OrcVole** [immich-app/immich#6781] [Feature Request]
### The feature

Currently, Immich does not let you sort the sequence in which you view images in an album. Also, it doesn't let you arrange your albums into an order you like.

Please support user assigned arrangement of images and albums. Hopefully, that won't be too difficult to do.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by LovelyCatEx** [comment]
I strongly support your suggestion. I've mentioned it in #6624, but closed.

**Comment by bo0tzz** [comment]
#2208
